.docker/lighttpd.conf

```diff
@@ -15,6 +16,8 @@ server.username             = "www-data"
 server.groupname            = "www-data"
 server.port                 = 80
 
+accesslog.filename          = "/var/log/lighttpd/access.log"
```

Comment:
If this were to go in `/var/www/var/log/lighttpd` would we be able to access it from the host instead of having to go into the container? Would that help at all?

Reply:
If we do that, should we move the `error.log` to that same location? I was putting the `access.log` in the same place as `error.log`.

Reply:
Oh yeah, that's a good idea. Do you think?

Reply:
I'd be inclined to leave it in the container. I think we might stumble on permissions issues. I tried moving the `error.log` and it gets unhappy.

Maybe we can tackle moving them at a later date and just add the `access.log` with this PR now?